%YAML:1.0
---
world2map_ext: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 1., 0., 0., 14771,
           0., 1., 0., 84757,
           0., 0., 1., -39,
           0., 0., 0., 1. ]

